src/hipercow/driver.py

```diff
@@ -12,3 +13,44 @@ def __init__(self, root: Root, **kwargs):
 
     def submit(self, task_id: str, root: Root) -> None:
         pass  # pragma: no cover
+
+
+def list_drivers(root) -> list[str]:
+    path = root.path / "hipercow" / "config"
+    return [x.name for x in path.glob("*")]
+
+
+def load_driver(root: Root, driver: str | None) -> HipercowDriver:
+    dr = _load_driver(root, driver)
+    if not dr:
```

Comment:
So here one could say...
`if not dr:= _load_driver(root, driver):`

(Not that I think one necessarily should - but one could...!)

Reply:
yeah, that would be a place one could use that!